pages/index.vue

```diff
@@ -139,6 +134,25 @@ export default defineComponent({
        * @caution you have to assign new value to todos for it to update
        */
     },
+    await this.$axios({
```

Comment:
Shouldn't this whole code be inside `getTasks()` method? The application is not running and throwing an error on the same line.